lib/App/Dochazka/REST/Guide.pm

```diff
@@ -517,10 +517,11 @@ sent.
 This section describes the C<App::Dochazka::REST> data model. Conceptually, 
 Dochazka data can be seen to exist in the following classes of objects:
 
-=over
-
 ##=item * Policy (parameters set when database is first created)
 ##
+
+=over
+
```

Comment:
@manwar Thanks for contributing. I guess you ran these files through `podchecker`? (Good idea!)

In this case, I'd rather have the commented-out `=item` be included in the `=over . . . =back` block. Instead of moving it up like you did, could you move it down to the end of the list (i.e. between "Components") and `=back`)? That should make podchecker happy.